### PR TITLE
[SDK-2154] Update e2e tests

### PR DESCRIPTION
**Pull request by frederikprijck**
- Changed the tolerance check to avoid an edge case that randomly fails the integration tests
- Avoid using the Auth0 SDK directly, but use e2e test through the playground
- Wrap common playground functionalities in custom Commands
- Prefer using `should` over `then` to do `expects` as those are retried when needed, see https://docs.cypress.io/api/commands/then.html#Syntax.
- Removed any delays we have
- Update package.json script to run on windows

### CODE REVIEW

File: cypress/integration/loginWithRedirect.js [hunk 1]
```diff
@@ -37,26 +36,24 @@ describe('loginWithRedirect', function () {
...
+
+    cy.url().should(url => {
+      const { scope } = decode(new URL(url).search.substr(1));
+      shouldBe(scope, 'openid profile email test test2');
     });
   });
 
   it('can perform the login flow', () => {
-    whenReady().then(win => {
+    whenReady().then(() => {
```
Comment (stevehobbsdev) [inline cypress/integration/loginWithRedirect.js:51]: Just an observation - why does this call to `whenReady` without a param pipe to `then` but the others don't?

**Review by stevehobbsdev — APPROVED** [review]
Very nice indeed, looks much cleaner 👏 👏

File: cypress/integration/loginWithRedirect.js (hunk 1 above)
Comment (frederikprijck) [reply cypress/integration/loginWithRedirect.js:51]: I use `should` for when doing expects and `then` for when just chaining things. In this particular case, there is no `expect` inside the `then`.
Except on line 56, there should did not work which i do not fully understand but then does work in that case.
